indico/util/placeholders.py

```diff
@@ -38,6 +42,31 @@ class Placeholder(object):
     required = False
     #: A short description of the placeholder.
     description = None
+    #: Whether the placeholder should not be shown by default
+    advanced = False
+
+    @classproperty
+    @classmethod
+    def friendly_name(cls):
+        return '{%s}' % cls.name
```

Comment:
Weird, why didn't I get an email for this comment?

Anyway, it's because `'{{{}}}'.format(cls.name)` is much less readable